### PR TITLE
mutt command error check

### PR DESCRIPTION
**Pull request by DeepakChaurasia30**
* Add error checking for the mutt command to make sure the email was sent .

### CODE REVIEW

File: email/mass_mailer [hunk 1]
```diff
@@ -18,6 +18,13 @@ while read -r address
 do
 	echo "sending message for ${kernel_version} to $address"
 	mutt -s "$subject" "$address" <  $msg >> spam.log 2>&1
+
```
Comment (gregkh) [inline email/mass_mailer:21]: no need for extra blank line.

File: email/mass_mailer [hunk 2]
```diff
@@ -18,6 +18,13 @@ while read -r address
 do
 	echo "sending message for ${kernel_version} to $address"
 	mutt -s "$subject" "$address" <  $msg >> spam.log 2>&1
+	
+
+	if [ $? -ne 0 ]; then
+        echo "Error: failed to send email to $address"
```
Comment (gregkh) [inline email/mass_mailer:24]: indent properly.  And error out to the log we already use.

File: email/mass_mailer [hunk 3]
```diff
@@ -18,6 +18,13 @@ while read -r address
...
+        fi
+
```
Comment (gregkh) [inline email/mass_mailer:26]: again, no need for 2 blank lines.

File: email/mass_mailer (hunk 2 above)
Comment (gregkh) [reply email/mass_mailer:24]: Also, will mutt not already have written out the error to the log?

**Review by gregkh — CHANGES_REQUESTED** [review]

File: email/mass_mailer (hunk 1 above)
Comment (DeepakChaurasia30) [reply email/mass_mailer:21]: Yeah, sorry for it

File: email/mass_mailer (hunk 2 above)
Comment (DeepakChaurasia30) [reply email/mass_mailer:24]: Yeah ,better to Replaced the if [ $? -ne 0 ] with if ! mutt to simplify the code.

**Pull request closed without merging**